

pandas.errors.OutOfBoundsTimedelta#


exception pandas.errors.OutOfBoundsTimedelta#
Raised when encountering a timedelta value that cannot be represented.
Representation should be within a timedelta64[ns].
Examples
>>> pd.date_range(start="1/1/1700", freq="B", periods=100000)
Traceback (most recent call last):
OutOfBoundsTimedelta: Cannot cast 139999 days 00:00:00
to unit='ns' without overflow.




